internal/pkg/service/appsproxy/dataapps/appconfig/appconfig_test.go

```diff
@@ -70,13 +77,10 @@ func TestLoader_LoadConfig(t *testing.T) {
 				{
 					responses: []*http.Response{
 						newResponse(t, 500, map[string]any{}, "", ""),
-						newResponse(t, 200, map[string]any{"upstreamAppUrl": "http://app.local"}, `"etag-value"`, "max-age=60"),
+						newResponse(t, 200, appPayload, `"etag-value"`, "max-age=60"),
 					},
-					expectedConfig: api.AppConfig{
-						ID:             "test",
-						UpstreamAppURL: "http://app.local",
-					},
-					expectedModified: true,
+					expectedErrorCode: 0, // no error expected
+					expectedModified:  true,
```

Comment:
All payloads in the test are same, I replaced the check with `assert.NotEmpty`, it is test for caching, not for JSON parsing.